src/ui.rs

```diff
@@ -47,14 +51,15 @@ pub struct TouchStickUiBundle<S: StickIdType> {
     pub view_visibility: ViewVisibility,
     /// Indicates the depth at which the node should appear in the UI
     pub z_index: ZIndex,
+    /// cursor position relative too the `TouchStick` in normalized logical pixels
     pub cursor_pos: RelativeCursorPosition,
 }
 
-/// bevy ui config for a stick
+/// marker component containing `TouchStickId`
```

Comment:
I'm not really sure what this component is for anymore. Can we remove it? In any case, "marker" components are generally unit structs (without fields).

Reply:
It can definitely be removed.
I only called it a marker component because of how it appeared to be used.

i did not want too add or remove types/fn with this commit, but if that is acceptable i can remove it